api/src/main/java/org/pmiops/workbench/config/ElasticSearchConfig.java

```diff
@@ -0,0 +1,29 @@
+package org.pmiops.workbench.config;
+
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.inject.Provider;
+
+@Configuration
+public class ElasticSearchConfig {
+
+  @Autowired
+  private Provider<WorkbenchConfig> configProvider;
+
+  //Is Singleton scope appropriate here? or should we use another scope for elasticsearch?
+  //If a scope beside Singleton is used we need to make sure we close this client when finished.
+  @Bean
+  public RestHighLevelClient client() {
+    HttpHost httpHost = new HttpHost(
```

Comment:
Could you also add a feature flag to the config in this PR? e.g. enableElasticsearchBackend - then just throw an exception or return null here if that's not on. Want to make sure this isn't crashing in higher environments where we don't have ES yet.

Reply:
Done.